ydb/core/kqp/opt/kqp_query_plan.cpp

```diff
@@ -515,17 +515,23 @@ class TxPlanSerializer {
     }
 
     TString DescribeValue(const NKikimr::NClient::TValue& value) {
-        auto str = value.GetDataText();
-        switch (value.GetType().GetData().GetScheme()) {
-        case NScheme::NTypeIds::Utf8:
-        case NScheme::NTypeIds::Json:
-        case NScheme::NTypeIds::String:
-        case NScheme::NTypeIds::String4k:
-        case NScheme::NTypeIds::String2m:
-            return "«" + str + "»";
-        default:
-            return str;
+        if (value.GetType().GetKind() == NKikimrMiniKQL::ETypeKind::Data) {
+            auto str = value.GetDataText();
+            switch (value.GetType().GetData().GetScheme()) {
+            case NScheme::NTypeIds::Utf8:
+            case NScheme::NTypeIds::Json:
+            case NScheme::NTypeIds::String:
+            case NScheme::NTypeIds::String4k:
+            case NScheme::NTypeIds::String2m:
+                return "«" + str + "»";
+            default:
+                return str;
+            }
+        }
+        if (value.GetType().GetKind() == NKikimrMiniKQL::ETypeKind::Pg) {
+            return value.GetPgText();
```

Comment:
Why without «»?

Reply:
PgText is quoted, hence we do not need second quote pair